app/local.env

```diff
@@ -39,7 +39,7 @@ LOG_ENABLE_AUDIT=FALSE
 DB_HOST=main-db
 DB_NAME=app
 DB_USER=app
-DB_SCHEMA=public
+DB_SCHEMA=app
```

Comment:
Since we're no longer using the default `public` schema, `make init-db` needs to set up the schema via (`make init-schema`).